awscli/clidriver.py

```diff
@@ -656,10 +657,15 @@ def invoke(self, service_name, operation_name, parameters, parsed_globals):
             value is returned.
 
         """
+        if hasattr(parsed_globals, 'read_timeout'):
```

Comment:
If I don't do it this way, tests fail:

`AttributeError: 'Namespace' object has no attribute 'read_timeout'`